bootstrap/bootstrap.py

```diff
@@ -113,35 +122,64 @@ def ingest():
+
+        input_dataset_id = get_dataset_id(create_dataset_url, copy.deepcopy(headers_for_ingestion), input_dataset_title, input_schema_id, data_for_dataset)
+
+        batch_id = get_batch_id(create_batch_url, copy.deepcopy(headers_for_ingestion), input_dataset_id, data_for_batch)
+
         replace_tenant_id(original_file, file_with_tenant_id, tenant_id)
-        upload_file(create_batch_url, headers, file_with_tenant_id, input_dataset_id, batch_id)
-        close_batch(create_batch_url, headers, batch_id)
-        if is_output_schema_different:
-            output_mixin_id = get_mixin_id(create_mixin_url, headers, output_mixin_title, data_for_output_mixin,
+
+        upload_file(create_batch_url, copy.deepcopy(headers_for_ingestion), file_with_tenant_id, input_dataset_id, batch_id)
+
+        close_batch(create_batch_url, copy.deepcopy(headers_for_ingestion), batch_id)
+
+        if is_output_schema_different == "True":
+            output_mixin_id = get_mixin_id(create_mixin_url, copy.deepcopy(headers_for_ingestion), output_mixin_title, data_for_output_mixin,
                                            class_id, tenant_id, output_mixin_definition_title)
-            output_schema_id = get_schema_id(create_schema_url, headers, output_schema_title, class_id, output_mixin_id,
+            output_schema_id = get_schema_id(create_schema_url, copy.deepcopy(headers_for_ingestion), output_schema_title, class_id, output_mixin_id,
                                          data_for_schema)
-            get_dataset_id(create_dataset_url, headers, output_dataset_title, output_schema_id, data_for_dataset)
+            get_dataset_id(create_dataset_url, copy.deepcopy(headers_for_ingestion), output_dataset_title, output_schema_id, data_for_dataset)
+
 
-    except requests.exceptions.HTTPError as httperr:
-        LOGGER.error('HTTPError Error: %s', httperr)
+    except requests.exceptions.HTTPError as http_err:
+        LOGGER.error('HTTPError Error: %s', http_err)
 
-    except requests.exceptions.ConnectionError as connerr:
-        LOGGER.error('ConnectionError Error: %s', connerr)
+    except requests.exceptions.ConnectionError as conn_err:
+        LOGGER.error('ConnectionError Error: %s', conn_err)
 
-    except requests.exceptions.Timeout as touterr:
-        LOGGER.error('Timeout Error: %s', touterr)
+    except requests.exceptions.Timeout as tout_err:
+        LOGGER.error('Timeout Error: %s', tout_err)
 
-    except requests.exceptions.RequestException as rerr:
-        LOGGER.error('Request Exception Error: %s', rerr)
+    except requests.exceptions.RequestException as r_err:
+        LOGGER.error('Request Exception Error: %s', r_err)
 
 
 if __name__ == "__main__":
-    ingest()
+    ingest_data = dictor(cfg, PLATFORM + ".ingest_data", checknone=True)
+    build_artifacts = dictor(cfg, PLATFORM + ".build_recipe_artifacts", checknone=True)
+
+    if ingest_data == "True":
+        LOGGER.debug("Ingesting data")
```

Comment:
Not familiar enough w/ the logging schema here but what I've seen in many cases is "info" level logging for breadcrumb tracking (e.g. "inside method a", "inside method b", etc...)  Then "debug" level provides additional details useful for figuring out what went wrong.  "Ingesting data" seems like info level to me.